packages/ember-data/lib/system/model/attributes.js

```diff
@@ -24,14 +24,16 @@ Model.reopenClass({
     Example
 
     ```javascript
+    import Ember from 'ember';
+    import DS from 'ember-data';
 
-    App.Person = DS.Model.extend({
+    var Person = DS.Model.extend({
       firstName: attr('string'),
       lastName: attr('string'),
       birthday: attr('date')
     });
 
-    var attributes = Ember.get(App.Person, 'attributes')
+    var attributes = Ember.get(Person, 'attributes')
```

Comment:
not sure what to do in those cases, where we consume the class.
Either use the `app/models/person.js` header, either replace the class declaration by an import statement like `import Person from app/models/person` ?